lib/spack/spack/config.py

```diff
@@ -211,11 +212,17 @@ def __repr__(self):
 _platform = spack.architecture.platform().name
 
 """Default configuration scope is the lowest-level scope. These are
-   versioned with Spack and can be overridden by sites or users."""
+   versioned with Spack and can be overridden by systems, sites or users."""
```

Comment:
Multi-line comments should go below the attribute they define, not above. See http://www.sphinx-doc.org/en/stable/ext/autodoc.html#directive-autoattribute:

> For module data members and class attributes, documentation can either be put into a comment with special formatting (using a #: to start the comment instead of just #), or in a docstring after the definition.

As it is now, each docstring will be associated with the previous line in our API docs. This is probably broken in many places in Spack.